libs/ui-watt/src/lib/components/input/date-range-input/watt-date-range-input.spec.ts

```diff
@@ -62,165 +53,180 @@ describe('Date range input - Reactive Forms', () => {
+      const { endDateInput } = await setup(withFormControl);
+      const pastedDate = completeDate.split('').reverse().join('');
+      const expectedDate = completeDate;
 
-    userEvent.paste(startDateInput, pastedDate, clipboardEvent, {
-      initialSelectionStart: 0,
-      initialSelectionEnd: 0,
-    });
+      const clipboardEvent: ClipboardEventInit = new Event('paste', {
+        bubbles: true,
+        cancelable: true,
+        composed: true,
+      });
 
-    expect(startDateInput).toHaveValue(expectedDate);
-  });
+      clipboardEvent.clipboardData = {
+        getData: () => pastedDate,
+      } as unknown as DataTransfer;
 
-  it('should be able to paste `yyyy-mm-dd` format into end date', async () => {
-    const { endDateInput } = await setup();
-    const pastedDate = completeDate.split('').reverse().join('');
-    const expectedDate = completeDate;
+      userEvent.paste(endDateInput, pastedDate, clipboardEvent, {
+        initialSelectionStart: 0,
+        initialSelectionEnd: 0,
+      });
 
-    const clipboardEvent: ClipboardEventInit = new Event('paste', {
-      bubbles: true,
-      cancelable: true,
-      composed: true,
+      expect(endDateInput).toHaveValue(expectedDate);
     });
 
-    clipboardEvent.clipboardData = {
-      getData: () => pastedDate,
-    } as unknown as DataTransfer;
+    it('should jump to end date, when typing in start date, and start date is complete', async () => {
+      const { startDateInput, endDateInput } = await setup(withFormControl);
+      const completeDateAndMore = completeDate.replace(/-/g, '') + '4';
+
+      userEvent.type(startDateInput, completeDateAndMore);
 
-    userEvent.paste(endDateInput, pastedDate, clipboardEvent, {
-      initialSelectionStart: 0,
-      initialSelectionEnd: 0,
+      expect(endDateInput).toHaveFocus();
     });
+  });
+
+  describe('with initial value', () => {
+    it('should have initial start and end date, if initial value is provided', async () => {
+      await setup(withInitialValue);
+      const initialDateRange = { start: completeDate, end: completeDate };
 
-    expect(endDateInput).toHaveValue(expectedDate);
+      expect(
+        screen.getByText(
+          `{ "start": "${initialDateRange.start}", "end": "${initialDateRange.end}" }`
+        )
+      ).toBeInTheDocument();
+    });
   });
 
-  it('should jump to end date, when typing in start date, and start date is complete', async () => {
-    const { startDateInput, endDateInput } = await setup();
-    const completeDateAndMore = completeDate.replace(/-/g, '') + '4';
+  describe('with validations', () => {
+    it('should not show error, before it has been touched', async () => {
+      await setup(withValidations);
 
-    userEvent.type(startDateInput, completeDateAndMore);
+      expect(screen.queryByText(/Field is required/i)).not.toBeInTheDocument();
+    });
+
+    it('should not show error, before it has been touched', async () => {
```

Comment:
suggestion: update test description. The test verifies that an error is shown when the component loses focus.